app/web/features/search/FilterDialog.tsx

```diff
@@ -212,15 +233,45 @@ export default function FilterDialog({
               />
 
               <Select
-                id="can_host_status_filter"
-                value={hostingStatusFilter}
-                onChange={(e) => {
-                  setHostingStatusFilter(e.target.value as number);
-                }}
+                id="can_host_status_filter_1"
+                variant="outlined"
                 label={t("search:form.host_filters.hosting_status_field_label")}
+                multiple={true}
+                menuItems={true}
```

Comment:
For boolean values that are true, you can just write `multiple` (without the ={true}` and it will work.